FPGA/core_rrv/de10_lite_core_rrv_top.sdc

```diff
@@ -9,6 +9,7 @@
 create_clock -period "10.0 MHz" [get_ports ADC_CLK_10]
 create_clock -period "50.0 MHz" [get_ports MAX10_CLK1_50]
 create_clock -period "50.0 MHz" [get_ports MAX10_CLK2_50]
+create_clock -period "15.0 KHz" [get_ports KBD_CLK]
```

Comment:
How this works.
I think there is an SDC file that needs to be aware of the KBD clock signal.. not sure here anything needs to be done..

Reply:
Quartus then throw message that its unconstrained clock